app/views/users/index.html.erb

```diff
@@ -12,7 +12,7 @@
   <thead>
     <th><%= User.human_attribute_name('name') %></th>
     <th><%= User.human_attribute_name('email') %></th>
-    <% if current_user.admin? %>
+    <% if current_user.admin? || Rails.env.development? %>
```

Comment:
Why do we check several times the admin role of the current user on this page, while this page is only accessible for admins (defined in the controller).